src/spu/src/smart_stream/memory.rs

```diff
@@ -10,12 +10,16 @@ const MEMORY: &str = "memory";
 
 /// Copy a byte array into an instance's linear memory
 /// and return the offset relative to the module's memory.
-pub fn copy_memory_to_instance(bytes: &[u8], instance: &Instance) -> Result<isize, Error> {
+pub fn copy_memory_to_instance(
+    store: &mut Store<()>,
```

Comment:
why need to pass store?

Reply:
In the new wasmtime API, a `&mut Store<T>` is required for calls to:

- `instance.get_memory`
- `instance.get_func`
- `alloc.call`
- `memory.data_ptr`

since all of this information is now tracked explicitly in a Store rather than implicitly through Rc inside these objects